api/room/schema.py

```diff
@@ -29,12 +29,41 @@ def mutate(self, info, **kwargs):
 
         return CreateRoom(room=room)
 
+class UpdateRoom(graphene.Mutation):
+
+    class Arguments:
+        room_id = graphene.Int()
+        room_type = graphene.String()
+        capacity = graphene.Int()
+        name = graphene.String()
+        
+
+    room = graphene.Field(Room)
+    
+    def mutate(self, info, room_id, **kwargs):
```

Comment:
@shemogumbe. The changes requested for updating using the room_id have been implemented.